doc/api/assert.markdown

```diff
@@ -2,42 +2,130 @@
 
     Stability: 3 - Locked
 
-This module is used so that Node.js can test itself. It can be accessed with
-`require('assert')`. However, it is recommended that a userland assertion
-library be used instead.
+The `assert` module is for use with Node.js' own unit tests. While it can be
```

Comment:
The tendency to favor `Node.js'` over `Node.js's` is so strong that I don't even bother correcting it directly anymore and instead recommend rewording entirely to avoid the possessive. So, something like:

> The `assert` module is for use with unit tests in the source code for Node.js.